### ISSUE TITLE
Support for Interpolated Controller Names

### ISSUE THREAD

**Issue by andrewmcodes** [marcoroth/stimulus-lsp#21]
I have the following code in one of my views:

```erb
<div data-controller="<%= stimulus_controller %>">
  <%# ... %>
</div>
```

Which creates an error with the diagnostic code `stimulus.controller.invalid` 

> "stimulus_controller" isn't a valid Stimulus controller. Did you mean "app--javascript--shared--table"?Stimulus LSP(stimulus.controller.invalid)

My expectation in this situation would be that the LSP ignores interpolated ruby code whether for controllers, values, actions, etc. rather than being an invalid controller name.

I have a little experience with VS Code extensions so would be happy to pair on this if you let me know!

Thanks for all you're doing with this!

**Comment by marcoroth** [comment]
Hey @andrewmcodes, thanks for reporting this one!

Agreed, this shouldn't raise an error. We should be able to handle that in here:

https://github.com/marcoroth/stimulus-lsp/blob/928168563f3428c6f9992c638dd8b7b5e44dc1d6/server/src/diagnostics.ts#L97

More than happy to pair on this one if you like 🙌🏼

**Comment by marcoroth** [comment]
I just released 0.3.0 earlier today, which ships @Gert-JanPeeters' fix from #61. Would be curious to hear if that solves this issue, thank you!

**Comment by marcoroth** [comment]
Going to close this, feel free to re-open if you are still seeing something that doesn't seem right! Thank you!
